baseplate/frameworks/pyramid/__init__.py

```diff
@@ -332,7 +332,8 @@ def _on_new_request(self, event: pyramid.events.ContextFound) -> None:
         )
         span.set_tag("protocol", "http")
         span.set_tag("http.url", request.url)
-        span.set_tag("http.route", request.matched_route.pattern)
+        # not sure this is accurate, as this is based on pyramid events 
+        # span.set_tag("http.route", request.matched_route.name)
```

Comment:
Happy to remove this but harold does something funky here that makes this not the Pyramid request object (and therefore there is no `pattern`. Considering this block is talking about events, I'm not sure this is in reaction to an actual request.

Reply:
This should be removed, it was added for the previous implementation that used spans. This is `endpoint = request.matched_route.pattern if request.matched_route else ""` in prom now btw.